### ISSUE TITLE
NYC support

### ISSUE THREAD

**Issue by chge** [artberri/rollup-plugin-istanbul#16]
Current version of [Istanbul](https://gotwarlost.github.io/istanbul/) is not actively mantained [anymore](https://github.com/gotwarlost/istanbul/issues/706).

[NYC](https://github.com/istanbuljs/nyc) contains [next version of Istanbul](https://istanbul.js.org/), that has API incompatible with this handy plugin, unfortunately.

My workaround is:
```javascript
import istanbul from 'rollup-plugin-istanbul';
import NYC from 'nyc';

const nyc = new NYC(),
      istanbul2 = new nyc._instrumenterLib.istanbul();

...

plugins: [
    istanbul({
        instrumenter: {
            Instrumenter: istanbul2.createInstrumenter
        }
    })
]
```

**Comment by jlmakes** [comment]
Nice workaround. The new Istanbul 2.0 ecosystem is a little confusing, since the package was split up into modules.

First class ES2015 support sounds great, and I hope proper support for source maps. Upgrading to the Istanbul 2.0 seems a worthy pursuit.

**Comment by artberri** [comment]
Sorry for being late with this issue @chge . New istanbul version is already implemented now in v2.0.0.
